packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.tsx

```diff
@@ -132,39 +102,48 @@ export type ColumnIconConfigs = {
   }
 }
 
-export type CommonCardProps = {
-  genericCardSchema?: GenericCardSchema
-  secondaryLabelLimit?: number
-  titleLinkConfig?: CardLink
-  ctaLinkConfig?: CTACardLink
-  labelLinkConfig?: LabelLinkConfig
-  descriptionConfig?: DescriptionConfig
-  rgbIndex?: number
-  columnIconOptions?: ColumnIconConfigs
-  releaseCardConfig?: ReleaseCardConfig
+export type QueryOrDeprecatedSearchParams = {
```

Comment:
Created this type to facilitate migrating towards just passing a query, and translating search params to query in the application